python/tests/test_ops.py

```diff
@@ -1619,6 +1619,22 @@ def test_divmod(self):
                     np.allclose(np_out[0], mx_out[0]), msg=f"Shapes {s1} {s2}, Type {t}"
                 )
 
+    def test_tile(self):
+        self.assertCmpNumpy([(2,), [2]], mx.tile, np.tile)
+        self.assertCmpNumpy([(2, 3, 4), [2]], mx.tile, np.tile)
+        self.assertCmpNumpy([(2, 3, 4), [2, 1]], mx.tile, np.tile)
```

Comment:
Could you add the case where the tiling dims are larger than the array dims? I don't think the code handles it properly now.

Reply:
added the test cases and the code to handle it